VideoLocker/res/layout-land/player_controller.xml

```diff
@@ -1,242 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
we're good to go i guess, my recent commit yields a good UX
https://github.com/edx/edx-app-android/pull/392#issue-111217097